### PR TITLE
Analytics event: GET_MEDIA

### LINKED ISSUE #1080: Analytics event: `GET_MEDIA`
## Description

The goal is to 

- add the type for event `GET_MEDIA` to the `Events` type.
  https://github.com/WordPress/openverse/blob/839418fabb0972ec3c3ccf48c72fd64cbba03cb9/frontend/src/types/analytics.ts#L12

- fire this event from the appropriate location in the codebase using the `useAnalytics` composable.
  https://github.com/WordPress/openverse/blob/839418fabb0972ec3c3ccf48c72fd64cbba03cb9/frontend/src/composables/use-analytics.ts#L11

## Additional information

- [RFC](https://github.com/WordPress/openverse/blob/main/rfcs/analytics/20221006-implementation_plan_frontend.md#events)
- [Example usage in `VHomeGallery.vue`](https://github.com/WordPress/openverse/blob/839418fabb0972ec3c3ccf48c72fd64cbba03cb9/frontend/src/components/VHomeGallery/VHomeGallery.vue#L128-L134)

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Fixes #1080 by @dhruvkb

## Description

This PR adds the `GET_MEDIA` analytics event that is fired when the user clicks the "Get this image" or "Get this audio" CTA button on the single result page.

## Testing Instructions

Set up the analytics events using `just frontend/up` and `just frontend/init`.
Run the app using `just frontend/dev`.
Click on the "Get this image" or "Get this audio"

Screenshot 2023-04-12 at 10 56 40 AM
Screenshot 2023-04-12 at 10 56 56 AM

Check that the event is shown in Plausible: http://0.0.0.0:50288/localhost?goal=GET_MEDIA

Screenshot 2023-04-12 at 10 56 06 AM

## Checklist

- [x] My pull request has a descriptive title (not a vague title like
      `Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or
      a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible
      errors.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: frontend/src/types/analytics.ts [hunk 1]
```diff
@@ -23,6 +25,19 @@ export type Events = {
     /** the identifier of the image */
     identifier: string
   }
+  /**
+   * Description: The user clicks the CTA button to the external source to use the image
+   * Questions:
+   *   - How often do users go to the source after viewing a result?
```
Comment (dhruvkb) [inline frontend/src/types/analytics.ts:31]: This is a bit inconsistent with the previous event, but it's better, so could you update the docstring of `CLICK_HOME_GALLERY_IMAGE` to match this?

File: frontend/src/types/analytics.ts [hunk 2]
```diff
@@ -23,6 +25,19 @@ export type Events = {
...
+   */
+  GET_MEDIA: {
+    /** The unique ID of the media */
```
Comment (dhruvkb) [inline frontend/src/types/analytics.ts:34]: Nit: since these aren't sentences...

```suggestion
    /** the unique ID of the media */
```

**Review by dhruvkb — APPROVED** [review]
LGTM! I have a few nits but nothing blocking.

**Review by zackkrida — APPROVED** [review]
Fantastic, LGTM.

**Pull request merged**